src/esn.calendar.libs/app/services/calendar-visibility-service.js

```diff
@@ -23,6 +25,34 @@ require('../app.constants.js');
       });
     }
 
+    function showOneCalendarAndHideOthers(calendar, status) {
+      var calId = calendar.getUniqueId();
+
+      storage.setItem(calId, status).then(function(hidden) {
+        $rootScope.$broadcast(CAL_EVENTS.CALENDARS.TOGGLE_VIEW, {
+          calendarUniqueId: calId,
+          hidden: status
+        });
+
+        return hidden;
```

Comment:
I think the return value is not used here, so not needed (the `hidden` parameter will not be needed then)